README.md

```diff
@@ -1,24 +1,156 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Budget-tracking App <a name="about-project"></a>
 
-* System dependencies
+[Budget-app] is a classic example of a blog website. It is a functional website that show the list of Categories and associated operations to each categories.
+[video description](https://drive.google.com/file/d/1KABOSfUWQScLh13Mcy4NtJGdfzoqvef7/view?usp=sharing)
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+The Recipe App was built with:
 
-* Database initialization
+    - Ruby On Rails
+    - PostgreSQL
+    - Html
+    - Tailwindcss
+    - Javascript
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Client</summary>
+</details>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+</details>
 
-* ...
+<details>
+<summary>Database</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add Categories**
+- **Add Transactions**
+- **Display transactions**
+- **Signup signin**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+To run this project, you'll need to install:
+
+    - Ruby 2.7.0 or later
+    - Rails 6.0 or later
+    - PostgreSQL
+
+### Installation
+
+Install this project with:
+
+```
+    Clone this repository to your desired folder:
+
+`git@github.com:juaryimami/gudget-app.git`
+
+cd recipe-app
+
+    Install the required gems:
+
+bundle install
+
+    Set up the database:
+
+rails db:create
+
+rails db:migrate
+```
+
```

Comment:
- [ ] It will be great to ensure that all the requirements for running your project locally are highlighted here. When I tried to run your project locally, I encountered an error `ActionView::Template::Error (The asset "tailwind.css" is not present in the asset pipeline.` It was only after googling how to solve this error that I found the solution as running `rake assets:precompile`. Kindly add this to the readme just incase someone else has the same problem